README.md

```diff
@@ -112,3 +112,52 @@ You can also set the stream options by using the following environment variables
 - `INTERFACE_NAME`
 - `INTERVAL_BTW_SAMPLES`
 - `SCALE`
+
+## Docker
+
+### Build the Container
+
+First, ensure you have Docker installed on your machine, then:
+1. Clone the repository containing the Stream Rust Test code
+2. Navigate to the root directory of the repository
+3. Run the build script to create the Docker image:
+   ```sh
+   ./scripts/docker/build.sh
+   ```
+
+### Run the Container
+
```

Comment:
I would like to improve the description that we have two different run options. What do you think?

```
#### Run the container with your configuration file.
```

```
#### Run the container with environment variables.
````

Reply:
Yes I think it's a good idea